website/components/Mermaid.tsx

```diff
@@ -0,0 +1,19 @@
+import type { CSSProperties, PropsWithChildren } from 'react';
+import './Mermaid.scss';
+
+interface Props {
+  style?: CSSProperties;
+  title?: string;
+}
+export default function Mermaid({
+  style,
+  children,
+  title,
+}: PropsWithChildren<Props>) {
+  return (
+    <div style={style} className="rspack-mermaid">
```

Comment:
Add `<NoSSR>` because it component is not SSR-friendly:

![image](https://github.com/user-attachments/assets/ad33da44-01c5-4a6f-8a9f-749365a2a4cc)

https://rspress.dev/api/client-api/api-components#nossr